demo_exam/ExamTasks.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+int countSymbols(char myChar[])
+{
+	int index = 0;
+	int counter = 0;
+	while (myChar[index] != '\0')
+	{
+		counter++;
+		index++;
+	}
+	return counter;
+}
+
+bool CheckDate(char date[])
+{
+	if (countSymbols(date) != 5)
+	{
+		return false;
+	}
+	if (date[2] != '.')
+	{
+		return false;
+	}
+	int myDate = (date[0] - '0') * 10 + date[1] - '0';
+	int month = (date[3] - '0') * 10 + date[4] - '0';
+	
+	if (month >= 1 && month <= 12 && myDate >= 1 && myDate <= 31)
+	{
+		if (month % 2 == 1 && month <= 7 && month != 2)
```

Comment:
The && month != 2 part here is not needed - you are checking for 'odd' months and 2 is an even number.